### ISSUE TITLE
Update - Query parameter mismatch at success mail

### ISSUE THREAD

**Issue by nupplaphil** [friendica/friendica#6772]
### Expected behavior

If an update is finished, the updater should send an email with https://github.com/friendica/friendica/blob/886d039dfe9c30ade8dde80b9e6563f6daff8f0d/src/Core/Update.php#L248

### Actual behavior

This is my output after the last update:
```console
[...]
2019-02-27 18:52:40 worker [INFO]: Update from '1301'  to '1303' - successful [] {"file":"Update.php","line":126,"function":"run","uid":"5f72292","process_id":16376}
2019-02-27 18:52:40 worker [WARNING]: Query parameters mismatch. {"query":"SELECT `uid`, `language`, `email` FROM `user` WHERE (`email` IN (%s))","args" {"1":"'admin@philipp.info'"},"callstack":"DBA::select, Update::updateSuccessfull, Update::run, Update::check"} {"file":"DBA.php","line":428,"function":"p","uid":"5f72292","process_id":16376}
2019-02-27 18:52:40 worker [INFO]: DB Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '%s))' at line 1 DBA::select, Update::updateSuccessfull, Update::run, Update::check SELECT `uid`, `language`, `email` FROM `user` WHERE (`email` IN (%s)) [] {"file":"DBA.php","line":569,"function":"p","uid":"5f72292","process_id":16376}
2019-02-27 18:52:40 worker [NOTICE]: Database structure update successful. [] {"file":"Update.php","line":280,"function":"updateSuccessfull","uid":"5f72292","process_id":16376}
[...]
```

I see the source:
https://github.com/friendica/friendica/blob/886d039dfe9c30ade8dde80b9e6563f6daff8f0d/src/Core/Update.php#L251-L252
But I don't get it why

### Steps to reproduce the problem

- set the `build` version down
- a worker should automatically trigger `DBUpdate`

### Friendica version you encountered the problem

2019.03-RC

### Friendica source (git, zip)

git

### PHP version

```
PHP 7.1.26 (cli) (built: Feb  6 2019 03:10:08) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.1.26, Copyright (c) 1999-2018, by Zend Technologies
```

### SQL version

```
mysql  Ver 15.1 Distrib 10.3.12-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
```

**Comment by nupplaphil** [comment]
```mysql
SELECT `uid`, `language`, `email` FROM `user` WHERE (`email` IN ('admin@philipp.info'));
+-----+----------+--------------------+
| uid | language | email              |
+-----+----------+--------------------+
|   1 | en       | admin@philipp.info |
+-----+----------+--------------------+
1 row in set (0.000 sec)
```

**Comment by MrPetovan** [comment]
The query parameter placeholder is now `?`, not `%s` anymore.

Suggestion to replace the two lines:
```php
$adminlist = DBA::select('user', ['uid', 'language', 'email'], ['email' => explode(",", str_replace(" ", "", Config::get('config', 'admin_email'))]);
```

**Comment by nupplaphil** [comment]
ah, thanks! I think I would have searched a while ;-)

**Comment by MrPetovan** [comment]
Fixed by #6773
